src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/config.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import os
+from azure.cli.core._environment import get_config_dir
+from azure.cli.core._config import AzConfig
+from azure.cli.core.util import CLIError
+
+
+class SfConfigParser(object):
+    def __init__(self, config_path=None):
+        if not config_path:
+            config_path = os.path.join(get_config_dir(), "config")
+        self.az_config = AzConfig()
+        self.az_config.config_parser.read(config_path)
+
+    def no_verify_setting(self):
+        return self.az_config.get("servicefabric", "no_verify", fallback="False") == "True"
+
+    def ca_cert_info(self):
+        using_ca = self.az_config.get("servicefabric", "use_ca", fallback="False")
+        if using_ca == "True":
```

Comment:
Yes, but since we are the only ones who set these strings we expect with the casing that we set them.